<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-NUS113PD" failures="1" tests="1" name="Default test(failed)" time="0.047" errors="0" timestamp="2022-08-26T09:34:13 IST">
  <testcase classname="TestNG_Assert.Assert_fail" name="Tc1" time="0.011">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG_Assert.Assert_fail.Tc1(Assert_fail.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Tc1 -->
  <testcase classname="TestNG_Assert.Assert_fail" name="Tc2" time="0.0">
    <ignored/>
  </testcase> <!-- Tc2 -->
</testsuite> <!-- Default test(failed) -->
